ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/Ds3ClientHelpers.java

```diff
@@ -467,6 +469,44 @@ public static Ds3ClientHelpers wrap(final Ds3Client client, final int retryAfter
      */
     public abstract Iterable<Ds3Object> listObjectsForDirectory(final Path directory) throws IOException;
 
+
+    /**
+     * Defaults to the path delimiter of '/'
```

Comment:
I would add a description of what exactly this method is useful for.  Specifically, what's the use case for this method.